apps/src/code-studio/appRedux.js

```diff
@@ -0,0 +1,31 @@
+const SET_APP_LOAD_STARTED = 'header/SET_APP_LOAD_STARTED';
+const SET_APP_LOADED = 'header/SET_APP_LOADED';
```

Comment:
nit: these actions still use the "header/" prefix

Reply:
Thanks! Will do.